src/OpenTelemetry.Instrumentation.AWSLambda/README.md

```diff
@@ -1,5 +1,8 @@
 # AWS OTel .NET SDK for Lambda
 
+[![nuget](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.AWSLambda.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AWSLambda)
+[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.AWSLambda.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AWSLambda)
```

Comment:
```suggestion
[![NuGet download count badge](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.AWSLambda.svg)](https://www.nuget.org/stats/packages/OpenTelemetry.Instrumentation.AWSLambda?groupby=Version)
```

Reply:
@Oberon00 - what d o you think about this link https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AWSLambda#versions-body-tab?

Serves similar data, but I think there is easier way to find other tabs

Reply:
Probably it's also fine to keep the originally proposed links. I have no strong opinion here.

Reply:
I will change descriptions and keep links.